docs/user_guide/api/LudwigModel.md

```diff
@@ -1,5 +1,5 @@
 <!-- DO NOT EDIT DIRECTLY: Automatically generated by code_doc_autogen.py. -->
-# LudwigModel class [[source]](https://github.com/ludwig-ai/ludwig/blob/master/ludwig/api.py#L92)
+# LudwigModel class [[source]](https://github.com/ludwig-ai/ludwig/blob/master/ludwig/api.py#L93)
```

Comment:
Hey @justinxzhao , I ran the `python code_doc_autogen.py` which updated this file. If I need to revert this, let me know!

Reply:
Hey @martindavis, it turns out to be a compatibility issue between code_doc_autogen and the markdown formatter. I pushed a fix to master -- could you rebase and re-run `python code_doc_autogen.py` and the pre-commit errors should go away.